content/master/guides/troubleshoot-crossplane.md

```diff
@@ -107,7 +107,7 @@ kind: Provider
 metadata:
   name: provider-aws
 spec:
-  package: xpkg.crossplane.io/crossplane-contrib/provider-aws:v0.33.0
+  package: xpkg.crossplane.io/crossplane-contrib/provider-aws:v2.0.0
```

Comment:
this is the non-upjet provider-aws, it is not on v2.0 yet. Latest release from https://github.com/crossplane-contrib/provider-aws/releases appears to be `v0.54.2`